WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -278,6 +278,20 @@ def update_events
     end
   end
 
+  def update_wcif
+    @competition = competition_from_params
+    wcif = params.permit!.to_h
```

Comment:
I'm struggling to answer this question. The problem may be that I attempted to create an `update_wcif` endpoint, which reintroduces the the weirdness we've already struggled with of "what does it mean to update the wcif?". Do you think this would be better as a `update_events` endpoint?